### PR TITLE
adding options menu on prepack.io/repl.html

### PR DESCRIPTION
**Pull request by joeymrios**
This should complete work for: #958 
Made it easy to add new options down the line.

**Preview**
default
hover-clicked

### CODE REVIEW

**Comment by NTillmann** [comment]
This looks awesome! Thanks for doing this. +1 for @cblappert's request.

**Comment by joeymrios** [comment]
I'll make the OPTIONS button show a button border like save/delete, without showing it just on hover over.

**Comment by joeymrios** [comment]
done. what do you all think?

**Comment by NTillmann** [comment]
I like the look!

I just tried it out, and there's just one small wrinkle: For the `mathRandomSeed`, it matters whether it's `undefined`, or something else. Your change always makes it a string. This actually causes Prepack to generate different code for the following simple example program:

```js
z = Math.random();
```

Can you turn the empty string into `undefined`?

Thanks!

**Comment by joeymrios** [comment]
done

File: js/repl.js [hunk 1]
```diff
@@ -18,9 +18,37 @@ function createEditor(elem) {
...
+  {
+    type: "string",
+    name: "mathRandomSeed",
+    defaultVal: "",
+    description: "If you want Prepack to evaluate Math.random() calls, please provide a seed."
+  },
+  {
+    type: "boolean",
+    name: "inlineExpressions",
+    defaultVal: false,
```
Comment (hermanventer) [inline js/repl.js]: I would love to see true be the default value here.

Comment (NTillmann) [reply js/repl.js]: +1. That would fix #1007!

**Comment by yinghuitan** [comment]
Currently, each option takes three lines in visual space. Can you put all the checkboxes, input field and dropdown list to the same line as the option name? Then two lines will be enough.

File: js/repl.js (hunk 1 above)
Comment (joeymrios) [reply js/repl.js]: done

**Pull request merged**